pkg/apis/core/v1/types.go

```diff
@@ -821,6 +821,13 @@ type (
 
 		// The reference to function
 		FunctionReference `json:"functionref"`
+
+		// HTTP Method for trigger
+		Method string `json:"method,omitempty"`
```

Comment:
HTTP Method for trigger function call

Please check if we can use kubebuilder default annotation to set it to POST.
Also add default being POST in the CRD doc above.